_posts/2019-09-16-deep-racer.markdown

```diff
@@ -0,0 +1,80 @@
+
+There are a few strategies to help reduce the disconnect between the simulated and real environments:
+
+* Environment control: make the simulated environment representative of the real world, and vice versa, including the physics model.  This simulates dark track; blue sky, etc.  By considering how the physics in the simulator correspond to the real world, you can make the two as close as possible.  For example, if the real world track has a rough surface, how would that affect friction compared to the simulation.  40% throttle in the simulation may result in a speed closer to 37% in the real world.  By following the AWS recommendations for the [physical track](https://docs.aws.amazon.com/deepracer/latest/developerguide/deepracer-build-your-track.html) you are controlling the environment: making the real world as close to the simulated environment as possible.  The colour image from the camera is converted to greyscale before being passed to the deep racer model - if you choose the wrong colour for your physical barriers, you can end up with unexpected results.
+* Domain randomisation: alter elements of the simulation to prevent the model training for a specific configuration.  Without randomisation, you may end up training your model for something [unintended](https://becominghuman.ai/its-magic-i-owe-you-no-explanation-explainableai-43e798273a08?gi=ea64a7743133).  Deep Racer trains on [greyscale images](https://docs.aws.amazon.com/deepracer/latest/developerguide/deepracer-how-it-works.html), leading to a dark track, with light boundary and centre lines.  Models trained in the simulator will not perform well in a real world where the track was made on a white floor.  Shadows could add unexpected gradient differences that lead to unexpected actions.  By varying the environment, and training on a variety of track configurations, the model could be made to cater for a wider range of environments.  This requires much more training required, and one reason why real autonomous vehicles need vast amounts of training: to cope with the wide and wacky environmental differences experienced in the real world.
+* Modularity and abstraction: break up the learning model, and the problem, into discreet modules by building a [convolutional neural network](https://en.wikipedia.org/wiki/Convolutional_neural_network).  The CNN would be trained to identify a road, the centre line, etc.  The reinforcement learning model is then passed the classified images: "road", "not road", "left of centre", "right of centre", etc.  AWS Deep Racer follows this approach.  Rather than having to process the images directly, the reward function is passed parameters about the environment that have been derived from the incoming images.
+
+# Conclusion
+
+In this post, I've given a brief introduction into AWS Deep Racer and Reinforcement learning.  In a future post, I shall cover creating your first model.  
+
+See you at Latency!
```

Comment:
perhaps you could mention there are a few workshop tickets left (there is a deepracer workshop)